doc/01-running-spacegraphcats.md

```diff
@@ -25,19 +25,22 @@ You will have a bunch of new output files:
 
 * the `dory/` directory contains the BCALM assembly of the input files into a compact De Bruijn graph; the key file here is `dory/bcalm.dory.k21.unitigs.fa`. There is also an output log file, `bcalm.dory.k21.unitigs.fa.log.txt`, that contains the console output of BCALM's run.
 * the `dory_k21_r1/` directory contains the various files of the catlas constructed by spacegraphcats:
-    * cdbg.gxt - the cDBG connection graph, in a custom format
-    * contigs.fa.gz - the unitigs from the cDBG
-    * contigs.fa.gz.indices - a numpy savez file containing mapping arrays
-    * contigs.fa.gz.info.csv - summary information about the cDBG unitigs
-    * contigs.fa.gz.mphf - Minimal Perfect Hash Function parameters for all of the k-mers in the cDBG
-    * first_doms.txt - the dominating set information for the cDBG
-    * catlas.csv - the catlas for the cDBG
+    * cdbg.gxt - the cDBG connection graph, in a custom format; produced by `bcalm_to_gxt.py`
+    * reads.bgz.index - a BGZF file containing the cDBG unitigs; produced by `bcalm_to_gxt.py`
+    * contigs.info.csv - summary information about the cDBG unitigs; produced by `bcalm_to_gxt.py`
+    * contigs.indices - a numpy savez file containing mapping arrays; produced by `index_contigs_by_kmer.py`
+    * contigs.mphf - Minimal Perfect Hash Function parameters for all of the k-mers in the cDBG; produced by `index_contigs_by_kmer.py`
+    * contigs.sig - sourmash signature for cDBG nodes. Defaults to scaled = 1000.
+    * contigs.sizes - sizes of all cDBG nodes in pickle format; produced by `index_cdbg_by_kmer.py`
+    * first_doms.txt - the dominating set information for the cDBG; produced by `spacegraphcats.catlas.catlas`
```

Comment:
FYI, this would be `spacegraphcats/catlas/catlas.py` in keeping with the above style of script reference, but with full paths added.